### DISCUSSION TITLE
背景图片没有加载

### DISCUSSION THREAD

**Discussion by scutLMG** [ppoffice/hexo-theme-icarus#1063] [Q&A] [answered]
我修改了一些文件以添加背景图片，在hexo s 本地浏览时是加载出来了的，但hexo d之后的githubPage页面没有变化。
以下是我修改的文件。向`icarus\include\style\base.style`添加了如下代码，图片放在`blog\node_modules\hexo-theme-icarus\source\img`里
```
body
    background: url(../img/mount.jpg) no-repeat
    background-attachment: fixed
    background-size: cover
```
是修改有问题还是这两个加载位置不一样呀？该如何修改背景图片呢？
我看到本地预览加载，其他图片、logo的initiator是index，但背景图片是default.css，有关吗？我对前端不是很了解。

**Comment by scutLMG** [comment]
我发现好像我在 `blog\node_modules\hexo-theme-icarus\source\img` 内添加的img、front等等，在上传到github后都没有被加载

**Comment by scutLMG** [comment, marked as answer]
好吧，我又魔改了一大堆样式，部署之后发现完全没有改变，新部署的文章倒是有，然后我意识到有什么问题。
查了下shift+f5强制刷新之后解决了。 :cry: 💢

**Comment by Ysfun** [comment]
问题解决了吗，我也碰到类似的问题了，我部署到gitPage上正常，但是部署到个人服务器上背景图片却没有显示

**Reply by scutLMG** [reply]
上面我已经回复了啊，强制刷新，除非你的背景图片放的地方本来就是不对的

**Reply by Ysfun** [reply]
好的好的，我清楚浏览器缓存后恢复正常了
